core/src/main/kotlin/com/malinskiy/marathon/execution/policy/ScreenRecordingPolicy.kt

```diff
@@ -0,0 +1,28 @@
+package com.malinskiy.marathon.execution.policy
+
+import com.fasterxml.jackson.annotation.JsonCreator
+
+/**
+ * Defines when screen recordings should be kept.
+ */
+enum class ScreenRecordingPolicy {
+    /**
+     * Keep screen recording only if the test failed. (Default)
+     */
+    ON_FAILURE,
+    /**
+     * Keep screen recording in any case.
+     */
+    ON_ANY;
+
+    companion object {
+        @JvmStatic
+        @JsonCreator
+        fun fromString(key: String?): ScreenRecordingPolicy? {
+            return when (key) {
+                ON_ANY.name -> ON_ANY
+                else -> ON_FAILURE
+            }
```

Comment:
@Malinskiy what is your suggestion here? 

Just for me fallback look strange here. 
ON_ANY.name -> ON_ANY
ON_FAILURE.name -> ON_FAILURE
else -> throw Exception("Wrong screen recording policy. Use one of ${ScreenRecordingPolicy.values()}")

Reply:
Ok let me change this